src/components/Picklist/pageObject/index.js

```diff
@@ -0,0 +1,142 @@
+const PagePicklistOption = require('../../PicklistOption/pageObject');
+
+/**
+ * Picklist page object class.
+ * @class
+ * @tutorial picklist
+ */
+class PagePicklist {
+    /**
+     * Create a new PagePicklist page object.
+     * @constructor
+     * @param {string} rootElement - The selector of the PagePicklist root element.
+     */
+    constructor(rootElement) {
+        this.rootElement = rootElement;
+    }
+
+    /**
+     * Clicks the input element.
+     * @method
+     */
+    clickInput() {
+        $(this.rootElement)
+            .$('input[type="text"]')
+            .click();
+    }
+
+    /**
+     * Focus the input element.
+     * @mehod
+     */
+    focusInput() {
+        this.clickInput();
+        this.waitUntilOpen();
+        this.clickInput();
+    }
+
+    /**
+     * Returns true when the input element has focus.
+     * @method
+     * @returns {bool}
+     */
+    hasFocusInput() {
+        return $(this.rootElement)
+            .$('input[type="text"]')
+            .isFocused();
+    }
+
+    /**
+     * It moves the pointer over the menu scroll up arrow
+     * @method
+     */
+    hoverScrollUpArrow() {
+        return $(this.rootElement)
+            .$('div.rainbow-picklist_dropdown-arrow-button.rainbow-picklist_dropdown-arrow-up')
+            .moveTo();
+    }
+
+    /**
+     * It moves the pointer out of the menu scroll up arrow
+     * @method
+     */
+    mouseLeaveScrollUpArrow() {
+        return $(this.rootElement)
+            .$('input[type="text"]')
+            .moveTo();
+    }
+
+    /**
+     * It moves the pointer over the menu scroll down arrow
+     * @method
+     */
+    hoverScrollDownArrow() {
+        return $(this.rootElement)
+            .$('div.rainbow-picklist_dropdown-arrow-button.rainbow-picklist_dropdown-arrow-down')
+            .moveTo();
+    }
+
+    /**
+     * It moves the pointer out of the menu scroll down arrow
+     * @method
+     */
+    mouseLeaveScrollDownArrow() {
+        return $(this.rootElement)
+            .$('input[type="text"]')
+            .moveTo();
+    }
+
+    getLabel() {
```

Comment:
getSelectedOptionLabel because getLabel can be the input label and it is confusing